lineman/app/components/group_page/group_page.scss

```diff
@@ -59,12 +59,23 @@
   @include card;
 }
 
+.group-page__threads {
+  padding: 10px 0 17px;
+  h2 {
```

Comment:
Actually I take that back.  The h2 styling needs to apply to the group description and the discussion.  So in this case it needs to be nested.